spec/acceptance/splunk_forwarder_spec.rb

```diff
@@ -1,25 +1,31 @@
-require 'spec_helper_acceptance'
+# require 'spec_helper_acceptance'
 
-describe 'splunk::forwarder class' do
-  context 'default parameters' do
-    # Using puppet_apply as a helper
-    it 'works idempotently with no errors' do
-      pp = <<-EOS
-      class { '::splunk::forwarder': }
-      EOS
+# describe 'splunk::forwarder class' do
```

Comment:
isn't this spec file needed anymore, or did you disable it just for testing?

Reply:
Disabled just for testing

Reply:
Reenabled these since tests are now working inside a docker container.